fpm/after-install.sh

```diff
@@ -33,7 +33,6 @@ chmod 700 /var/lib/zeyple/keys
 
 chown zeyple: /var/lib/zeyple/keys
 
-cp zeyple.py /usr/local/bin/zeyple.py
```

Comment:
Just to be clear, you removed this line because `zeyple.py` is already being copied upon installation of the .deb package right?

Reply:
Yes